src/road_cmd.cpp

```diff
@@ -2098,14 +2098,12 @@ static void GetTileDesc_Road(TileIndex tile, TileDesc *td)
 	RoadType tram_rt = GetRoadTypeTram(tile);
 	if (road_rt != INVALID_ROADTYPE) {
 		const RoadTypeInfo *rti = GetRoadTypeInfo(road_rt);
-		td->roadtype = rti->strings.name;
-		td->road_speed = rti->max_speed / 2;
+		td->road_desc = TrackDesc(rti->strings.name, rti->max_speed / 2, rti->grffile[0]);
```

Comment:
Same as above for road, `ROTSG_GROUND` (2).

(Also applies to all other instances where road/tram GRF is queried).